plugin-parser/mlops-plugin-parser.cc

```diff
@@ -202,25 +161,38 @@ _parse_json (const gchar *json_path, mlsvc_json_type_e json_type, const gchar *a
       case MLSVC_JSON_RESOURCE:
         {
           const gchar *name = json_object_get_string_member (object, "name");
-          const gchar *path = json_object_get_string_member (object, "path");
           const gchar *desc = json_object_get_string_member (object, "description");
-          const gchar *clear = json_object_get_string_member (object, "clear");
+          JsonNode *path_node = json_object_get_member (object, "path");
+          JsonArray *path_array = NULL;
+          guint path_len;
 
-          if (!name || !path) {
-            _E ("Failed to get name or path from json file '%s'.", json_file);
+          if (!name) {
+            _E ("Failed to get name or path from MLSVC_JSON_RESOURCE.");
             continue;
           }
 
-          bool clear_old = (clear && g_ascii_strcasecmp (clear, "true") == 0);
+          path_len = 1U;
+          if (JSON_NODE_HOLDS_ARRAY (path_node)) {
+            path_array = json_node_get_array (path_node);
+            path_len = (path_array) ? json_array_get_length (path_array) : 0U;
+          }
 
-          /* Remove old resource from database. */
-          if (clear_old) {
-            /* Ignore error case. */
-            ml_agent_resource_delete (name);
+          if (path_len == 0U) {
+            _E ("Failed to get path array from resource json.");
+            continue;
           }
 
-          ret = ml_agent_resource_add (
+          for (guint pidx = 0; pidx < path_len; pidx++) {
+            const gchar *path;
+
+            if(path_array)
```

Comment:
indent rule `if (`

Reply:
this parser depends on tizen feature and cannot check the indent rule in ci.
I recommend using clang-format (e.g., clang-format-12 -i --style=file ./plugin-parser/mlops-plugin-parser.cc) on your host pc.

Reply:
thanks, updated it!